crocket/src/main/java/com/crocket/DrawPanel.java

```diff
@@ -0,0 +1,62 @@
+package com.crocket;
+import java.awt.*;
+import java.awt.event.KeyEvent;
+import java.awt.event.KeyListener;
+import java.awt.image.BufferedImage;
+
+import javax.imageio.ImageIO;
+import javax.swing.ImageIcon;
+import javax.swing.JLabel;
+import javax.swing.JPanel;
+
+
+public class DrawPanel extends JLabel implements KeyListener{
+    
+    //temp placement
+    Dimension screen = Toolkit.getDefaultToolkit().getScreenSize();
+    Image ballImg = Toolkit.getDefaultToolkit().getImage("crocket/assets/textures/Ball.png");
```

Comment:
remove getters

Reply:
They are temporary and will be removed when we have a proper model